src/scripts/clipperUI/components/previewViewer/previewComponentBase.tsx

```diff
@@ -70,8 +70,10 @@ export abstract class PreviewComponentBase<TState, TProps extends ClipperStatePr
 	}
 
 	private getPreviewSubtitle(): any {
-		let sourceUrlCitation = Localization.getLocalizedString("WebClipper.FromCitation")
-			.replace("{0}", this.props.clipperState.pageInfo ? this.props.clipperState.pageInfo.rawUrl : "");
+		let sourceUrlCitationPrefix = Localization.getLocalizedString("WebClipper.FromCitation")
+			.replace("{0}", ""); // TODO can we change this loc string to remove the {0}?
```

Comment:
I was concerned about just making this change to remove the `{0}` from `WebClipper.FromCitation` (on both the client and server) - are there any weird timing issues to be aware of from a localization standpoint?